manifests/yamls/app-sync-labels.yaml

```diff
@@ -0,0 +1,113 @@
+    kubectl label clusterrolebinding argo-rollouts-aggregate-to-view "app.kubernetes.io/managed-by=Helm"
+    kubectl annotate clusterrolebinding argo-rollouts-aggregate-to-view "meta.helm.sh/release-name=$RELEASE_NAME" "meta.helm.sh/release-namespace=devtroncd"
+    kubectl label clusterrole argo-rollouts-clusterrole "app.kubernetes.io/managed-by=Helm"
+    kubectl annotate clusterrole argo-rollouts-clusterrole "meta.helm.sh/release-name=$RELEASE_NAME" "meta.helm.sh/release-namespace=devtroncd"
+    kubectl label clusterrolebinding argo-rollouts-clusterrole "app.kubernetes.io/managed-by=Helm"
+    kubectl annotate clusterrolebinding argo-rollouts-clusterrole "meta.helm.sh/release-name=$RELEASE_NAME" "meta.helm.sh/release-namespace=devtroncd"
+    kubectl label clusterrole workflow-cluster-role "app.kubernetes.io/managed-by=Helm"
+    kubectl annotate clusterrole workflow-cluster-role "meta.helm.sh/release-name=$RELEASE_NAME" "meta.helm.sh/release-namespace=devtroncd"
+    kubectl label clusterrolebinding workflow-cluster-role "app.kubernetes.io/managed-by=Helm"
+    kubectl annotate clusterrolebinding workflow-cluster-role "meta.helm.sh/release-name=$RELEASE_NAME" "meta.helm.sh/release-namespace=devtroncd"
+    kubectl label clusterrole argo-ui-cluster-role "app.kubernetes.io/managed-by=Helm"
+    kubectl annotate clusterrole argo-ui-cluster-role "meta.helm.sh/release-name=$RELEASE_NAME" "meta.helm.sh/release-namespace=devtroncd"
+    kubectl label clusterrolebinding argo-ui-cluster-role "app.kubernetes.io/managed-by=Helm"
+    kubectl annotate clusterrolebinding argo-ui-cluster-role "meta.helm.sh/release-name=$RELEASE_NAME" "meta.helm.sh/release-namespace=devtroncd"
+    kubectl label sa argo -n argo "app.kubernetes.io/managed-by=Helm"
+    kubectl annotate sa argo -n argo "meta.helm.sh/release-name=$RELEASE_NAME" "meta.helm.sh/release-namespace=devtroncd"
+    kubectl label role argo-role -n argo "app.kubernetes.io/managed-by=Helm"
+    kubectl annotate role argo-role -n argo "meta.helm.sh/release-name=$RELEASE_NAME" "meta.helm.sh/release-namespace=devtroncd"
+    kubectl label rolebinding argo-binding -n argo "app.kubernetes.io/managed-by=Helm"
+    kubectl annotate rolebinding argo-binding -n argo "meta.helm.sh/release-name=$RELEASE_NAME" "meta.helm.sh/release-namespace=devtroncd"
+    kubectl label cm workflow-controller-configmap -n argo "app.kubernetes.io/managed-by=Helm"
+    kubectl annotate cm workflow-controller-configmap -n argo "meta.helm.sh/release-name=$RELEASE_NAME" "meta.helm.sh/release-namespace=devtroncd"
+    kubectl label deploy workflow-controller -n argo "app.kubernetes.io/managed-by=Helm"
+    kubectl annotate deploy workflow-controller -n argo "meta.helm.sh/release-name=$RELEASE_NAME" "meta.helm.sh/release-namespace=devtroncd"
+---
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: app-sync-labels
```

Comment:
change the name to something generic devtron-housekeeping

Reply:
Renamed to devtron-housekeeping